File: app-java/src/main/java/com/java/bilibili/ui/main/fragment/impl/MainImpl.java
package com.java.bilibili.ui.main.fragment.impl;

import com.java.bilibili.base.BasePresenter;
import com.java.bilibili.base.BaseView;

public class MainImpl {
    public interface View extends BaseView {
        void initUserInfo(String data);
    }

    public interface Presenter extends BasePresenter<View> {
        void getUserInfo(String id);
    }
}

File: app-kotlin/src/main/java/com/kotlin/bilibili/ui/main/fragment/impl/MainImpl.kt
package com.kotlin.bilibili.ui.main.fragment.impl

import com.kotlin.bilibili.base.BasePresenter
import com.kotlin.bilibili.base.BaseView

/**
 * Created by Administrator on 2017/11/13 0013.
 */
interface MainImpl {
    interface View : BaseView {
        fun initUserInfo(data: String)
    }

    interface Presenter : BasePresenter<View> {
        fun getUserInfo(id: String)
    }
}